test/embedding/embedding.c

```diff
@@ -169,6 +169,12 @@ int main()
         checked_eval_string("LocalModule.myapp()");
     }
 
+    {
+        // Main.include and Main.eval exist (#28825)
+        checked_eval_string("include(\"include_and_eval.jl\")");
```

Comment:
Looks like windows CI can't find this file from here.

Reply:
Ah, appveyor.yml is running the embedding test with a different working directory. I pushed a tweak to make that consistent.